### PR TITLE
[CONENCT][PYTHON][DOC] Document how to run the module of tests for Spark Connect Python tests

### PR DESCRIPTION
**Pull request by amaliujia**
### What changes were proposed in this pull request?

Improve developer documentation for Connect project for how to run `pyspark-connect` module which runs all existing Connect Python tests.

### Why are the changes needed?

Developer facing documentation improvement.

### Does this PR introduce _any_ user-facing change?

NO

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by amaliujia** [comment]
R: @zhengruifeng

File: connector/connect/README.md [hunk 1]
```diff
@@ -52,9 +52,15 @@ To use the release version of Spark Connect:
 ### Run Tests
 
 ```bash
+# Run a single Python test.
 ./python/run-tests --testnames 'pyspark.sql.tests.connect.test_connect_basic'
```
Comment (zhengruifeng) [inline connector/connect/README.md:56]:
```suggestion
# Run a single Python class.
./python/run-tests --testnames 'pyspark.sql.tests.connect.test_connect_basic'
# Run a single test case in a specific class:
./python/run-tests --testnames 'pyspark.sql.tests.connect.test_connect_basic SparkConnectTests.test_schema'
```

**Review by grundprinzip — APPROVED** [review]

**Comment by zhengruifeng** [comment]
merged into master

**Comment by zhengruifeng** [comment]
oh, I forgot to mention that we should have a SPARK-XXXXX title

**Review by HyukjinKwon — COMMENTED** [review]
LGTM2

**Comment by amaliujia** [comment]
@zhengruifeng I confirmed that for small doc change there is no need for a JIRA (that is why I didn't create one).

**Pull request closed without merging**